rfcs/text/0352-graph-type-system.md

```diff
@@ -1338,6 +1338,27 @@ An **Entity Type** is a JSON schema that satisfies the following JSON meta-schem
     },
     "title": { "type": "string" },
     "description": { "type": "string" },
+    "default": {
+      "$comment": "Default Entity instance",
+      "type": "object",
+      "propertyNames": {
+        "$comment": "Property names must be a valid URI to a Property Type",
+        "type": "string",
+        "format": "uri"
+      }
+    },
+    "example": {
```

Comment:
Also, definitely not for now, but we should define things like `propertyNames` once in `$defs` instead of repeatedly throughout the schema.
```suggestion
    "examples": {
```